src/Resources/contao/helper/functions.php

```diff
@@ -15,9 +15,9 @@
  * @param string $strMessage
  * @param string $strLog
  */
-function log_message($strMessage, $strLog='error.log')
```

Comment:
And we cannot really use the container here anyway as the channels (target files) are already set and cannot be changed dynamically. So the only case where we could use the container here is if the container is built and ready to use and `$strLog` was not set explicitly.